inertia/http.py

```diff
@@ -143,7 +149,7 @@ def build_first_load(self, data):
                 "inertia_layout": layout,
                 **context,
             },
-            self.request,
+            self.request.request,
```

Comment:
This is a functional change. Was the code incorrect before?

Reply:
I wouldn't say it is incorrect, it works, but I'm not sure it can satisfy the type checker. `render_to_string` expects a `HttpRequest` from Django, and `self.request` is a InertiaRequest, which wraps the request it but is not a subclass from `HttpRequest`, so I got a type error. 

Because this is a Django function, it does not seem to me that it is accessing anything Inertia related, so I forwarded the inner request. Since the tests passed, this seemed correct to me, and the type checking error was gone. What do you think?

Reply:
I was just going to tag the same, thanks @sopelj . I'll see if we can get that PR merged asap.

Reply:
Just merged #84 so that should help us clean this up a bit!